micro-integrator/hello-world-mi/helloConfigs/ReadMe.html

```
@@ -0,0 +1,179 @@
+              <input type="checkbox" id="toggle2" class="toggle" />
+              <label class="title" for="toggle2"  
+                  style="border-bottom-color: rgba(68, 138, 255,.1); 
+                      background-color: rgba(68, 138, 255,.1); box-shadow: none !important;">
+                Run using external WSO2 Enterprise Integrator 6X Series
+              </label>
+              <div class="content" style="box-shadow: none !important;">
+                    <ol>
+                        <li>
+                            <h3>Connect the EI server with Integration Studio</h3>
+                            <p>If you haven't already attached the WSO2 EI runtime to WSO2 Integration Studio, follow the instructions given below.</p>
+                            <p>- Open the <strong>Integration Studio Getting Started</strong> and 
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p> ], start tag match failed [ <p> ] on line 125.

(tag-pair)